doc/api/process.md

```diff
@@ -1509,6 +1509,20 @@ tarball.
   - `'Argon'` for the 4.x LTS line beginning with 4.2.0.
   - `'Boron'` for the 6.x LTS line beginning with 6.9.0.
   - `'Carbon'` for the 8.x LTS line beginning with 8.9.1.
+* `majorVersion` {number} The major version of Node.js.
+* `minorVersion` {number} The minor version of Node.js.
+* `patchVersion` {number} The patch version of Node.js.
+* `prereleaseTag` {string} The SemVer pre-release tag for Node.js.
+* `compareVersion` {function} Perform a SemVer comparison to the release
```

Comment:
1. Nit: `{function}` -> `{Function}` (only primitives are lovercased).
2. We may need to add the function signature here, with all the optional parameters marked in `[]` (nested or not).
3. I am not sure if this is a common practice to not single out such API in its own section. Let us see what others think.